andela/bp-esa-frontend
# feat(Payload): update the frontend mock data to adapt to the backend payload

#### What does this PR do?
It updates the frontend mock data to adapt to what is expected from the backend data. It also fixes the codebase and tests to accommodate the new data.

#### Description of Task to be completed?
Ensure that the  mock data in the backend matches data coming from the backend.

#### How should this be manually tested?
- Clone and set up the project using the instruction in the README.
- $ git checkout to ft-163284732-update-frontend-to-adapt-to-backend-payload branch which contains the changes.
- Use $ npm start to start the application
- Visit http://localhost:3000/.
- Click on the automation modals to see the automation details
- Run the tests using `npm test`

#### Any background context you want to provide?
- The initial data from the backend did not match what was expected on the frontend.


#### What are the relevant pivotal tracker stories?
[#163284732](https://www.pivotaltracker.com/story/show/163284732)
